opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -23,7 +23,7 @@
 from google.api_core.gapic_v1 import client_info
 from google.cloud import monitoring_v3
 
-from opencensus.__version__ import __version__
+from opencensus.common.__version__ import __version__
```

Comment:
@c24t I wish to get your opinion as I was struggling with the ordering, should we put this below `from opencensus.common import utils` for alphabetic order?

Reply:
Also the file name, do we want just version.py? (personally I would prefer version.py than __version__.py, and version.py has been used in the official sample https://packaging.python.org/guides/single-sourcing-package-version/)
I did some research and seems there is no consistent pattern https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc_version.py.

Reply:
Got it, changing to version.py now.